package.nls.json

```diff
@@ -26,5 +26,7 @@
     "azureApiManagement.showArmProduct": "Edit Product",
     "azureApiManagement.openExtensionWorkspaceFolder": "Open extension workspace folder",
     "azureApiManagement.initializeExtensionWorkspaceFolder": "Initialize extension workspace folder",
-    "azureApiManagement.advancedPolicyAuthoringExperience": "(Experimental Feature) Enables advanced policy authoring experience."
+    "azureApiManagement.advancedPolicyAuthoringExperience": "(Experimental Feature) Enables advanced policy authoring experience.",
+    "azureApiManagement.extractApis": "Extract",
+    "azureApiManagement.extractSingleApi": "Extract"
```

Comment:
Should this be named as "azureApiManagement.extractApi": "Extract API ARM templates"